### PR TITLE
Adding GCM as an available Heapster sink.

### PR DESCRIPTION
**Pull request by vmarmol**

### CODE REVIEW

File: sinks/types.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 )
 
-var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery]")
+var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery | gcm ]")
```
Comment (vishh) [inline sinks/types.go]: Do we need any other auth information for gcm? What will the user have to do to get GCM to work?

**Comment by vishh** [comment]
LGTM.

File: sinks/types.go (hunk 1 above)
Comment (vmarmol) [inline sinks/types.go]: Today we only support running in GCE (which we check and error out on in New) and we get the auth information from the machine's service accounts (which we also check and error out on).

Comment (vishh) [inline sinks/types.go]: Ok. We can clarify that in the Readme as well.

**Pull request merged**